src/modules/previewpane/common/handlers/PreviewHandlerBase.cs

```diff
@@ -74,6 +75,7 @@ public static void Register(Type t)
         [ComUnregisterFunction]
         public static void Unregister(Type t)
         {
+            Console.WriteLine("Removing keys to registry.");

```

Comment:
this should be removed

Reply:
I put this as placeholder to display in shell when register / unregister function is called using `regasm`. Any specific reason to remove this?